src/common/src/telemetry/mod.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod manager;
+pub mod report;
+
+use std::time::SystemTime;
+
+use anyhow::{anyhow, Result};
+use serde::{Deserialize, Serialize};
+use sysinfo::{System, SystemExt};
+
+use crate::util::resource_util::cpu::total_cpu_available;
+use crate::util::resource_util::memory::{total_memory_available_bytes, total_memory_used_bytes};
+
+/// Url of telemetry backend
+pub const TELEMETRY_REPORT_URL: &str = "https://telemetry.risingwave.dev/api/v1/report";
+
+/// Telemetry reporting interval in seconds, 24h
+pub const TELEMETRY_REPORT_INTERVAL: u64 = 24 * 60 * 60;
```

Comment:
1day seems a bit too long for me. How about 3hr or 6hr?

Reply:
6hr LGTM